### ISSUE TITLE
[🐛 bug report] Environment map tear at -Z

### ISSUE THREAD

**Issue by michele-arrival** [mitsuba-renderer/mitsuba2#458]
## Summary

Rendering of an environment map shows a tear at -Z, where it shoud be continuous.

## Description
Only an environment map is rendered, with a perspective camera facing -Z
The rendered image shows a visible tear where the -X side of the envmap joins the +X side:

![image](https://user-images.githubusercontent.com/54891577/124286502-516e1400-db4f-11eb-9efa-7a93741c6271.png)

The environment map, when "wrapped around" in X and concatenated +X and -X shows no such tear
![image](https://user-images.githubusercontent.com/54891577/124287317-2801b800-db50-11eb-873e-c048794d0131.png)

![image](https://user-images.githubusercontent.com/54891577/124287506-60a19180-db50-11eb-93f0-d2a27bbdd596.png)

![image](https://user-images.githubusercontent.com/54891577/124287624-86c73180-db50-11eb-96f7-0c22d3bfd205.png)

## Steps to reproduce

*For bug report, please enumerate the steps to follow in order to reproduce the issue <- [remove this]*

1. scene.xml:
```
<scene version='2.2.1'>
    <integrator type="direct">
        <integer name="emitter_samples" value="4"/>
        <integer name="bsdf_samples" value="0"/>
    </integrator>

    <sensor type="perspective">
        <float name="near_clip" value="1"/>
        <float name="far_clip" value="1000"/>

        <transform name="to_world">
            <lookat target="0.0, 0.0, 0.0"
                    origin="0.0, 0.0, 3.5"
                    up    ="0.0, 1.0, 0.0"/>
        </transform>

        <film type="hdrfilm">
            <rfilter type="box"/>
            <integer name="width" value="640"/>
            <integer name="height" value="480"/>
        </film>

        <sampler type="independent">
            <integer name="sample_count" value="4"/>
        </sampler>
    </sensor>

    <emitter type="envmap">
		<string name="filename" value="grace-new.exr"/>
		<float name="scale" value="10.0"/>
	</emitter>
</scene>
```
2. envmap from https://vgl.ict.usc.edu/Data/HighResProbes/probes/grace-new.exr
3. run with `mitsuba -m packet_rgb scene.xml`

python script to show the wrapped/concatenated envmap:
```
import cv2
import imageio
import numpy as np


# imageio.plugins.freeimage.download()
img = imageio.imread('grace-new.exr') * 5
img = np.power(img, 1/2.2)
mid_y = img.shape[0] // 2
ims2 = 192
img_cat = np.concatenate((img[mid_y - ims2:mid_y + ims2, -ims2:, ::-1],
                          img[mid_y - ims2:mid_y + ims2,  :ims2, ::-1]), axis=1)

cv2.imshow("grace", img_cat)
cv2.waitKey(0)
```

## System configuration

- Platform: Windows 10
- Compiler: Visual Studio 2019 v16.10.2 x64 - Release
- Python version: None
- Mitsuba 2 version: 2.2.1
- Compiled variants:
    * `scalar_rgb`
    * `packet_rgb`
    * `gpu_rgb`
    * `gpu_autodiff_rgb`

**Comment by Speierers** [comment]
Thanks @michele-arrival for reporting this. This is a known issue, see PR #173.

I will close this issue as this PR should be merge in the near future.

**Comment by michele-arrival** [comment]
ooops, apologies I didn't see that. Thanks!

**Comment by michele-arrival** [comment]
If it's of any use knowing, the fix in #173 does indeed look good

![image](https://user-images.githubusercontent.com/54891577/124472031-eebe8780-dd9d-11eb-8be8-339aaf3d30d2.png)
